elyra/pipeline/pipeline.py

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import dataclasses
+from dataclasses import dataclass
```

Comment:
Should we update the `@dataclass` decorators to `@dataclasses.dataclass` and remove the second import, or remove the first import and extend the second with `is_dataclass, asdict` and remove the `dataclasses.` prefix from the static methods?

I'm inclined to go with the first approach since it is more descriptive.

Reply:
I suppose the first approach is contrary to "import only what is needed", so perhaps the import for `asdict` could be aliased to something like `asdict as dataclasses_asdict`.  `is_dataclass` seems fairly unique and self-describing.

Reply:
Yeah I had that thought re: the first approach as well. Will do!

Reply:
Whoops, I only saw your latest comment after I posted mine 😄 . Second approach it is, then